llvm/include/llvm/IR/DerivedTypes.h

```diff
@@ -492,9 +492,9 @@ class VectorType : public Type {
     return VectorType::get(EltTy, VTy->getElementCount());
   }
 
-  // This static method returns a VectorType with a smaller number of elements
-  // of a larger type than the input element type. For example, a <16 x i8>
-  // subdivided twice would return <4 x i32>
+  // This static method returns a VectorType with a larger number of elements
+  // of a smaller type than the input element type. For example, a <16 x i8>
```

Comment:
nit: I would change the example types to something more common like `<4 x i64>` instead of `<16 x i8>` which would get subdivided to `<16 x i16>`.

Same for the test?

Reply:
Done